include/fluent-bit/flb_input.h

```diff
@@ -497,9 +497,9 @@ static FLB_INLINE void input_pre_cb_collect(void)
     struct flb_coro *coro;
     struct flb_libco_in_params *params;
 
-    params = pthread_getspecific(libco_in_param_key);
+    params = (struct flb_libco_in_params *) pthread_getspecific(libco_in_param_key);
     if (params == NULL) {
-        params = flb_calloc(1, sizeof(struct flb_libco_in_params));
+        params = (struct flb_libco_in_params *) flb_calloc(1, sizeof(struct flb_libco_in_params));
```

Comment:
Could you please break this and other lines that exceed the 80 column limit?

Reply:
Hi, I just pushed some changes. I am not sure if I needed the backslash before the newline but it compiles.